cumulusci/robotframework/pageobjects/ObjectManagerPageObject.py

```diff
@@ -0,0 +1,239 @@
+        self.selenium.click_element(next_button)
+        self.selenium.wait_until_page_contains_element(formula_txtarea, 60)
+        self.selenium.get_webelement(formula_txtarea).send_keys(formula)
+        self.selenium.click_element(check_syntax)
+        self.selenium.click_element(next_button)
+        self.selenium.click_element(next_button)
+        self.selenium.click_element(save_button)
+        self.selenium.wait_until_location_contains(
+            "FieldsAndRelationships/view",
+            timeout=90,
+            message="Detail page did not load in 1 min",
+        )
+
+    def create_lookup_field(self, field_name, related):
+        """Creates a Lookpup field by taking in the inputs field_name and related field"""
+        option = object_manager["select_related_option"].format(related)
+        related = object_manager["select_related"].format("DomainEnumOrId")
+        self.selenium.wait_until_page_contains_element(lookup_locator, 60)
+        self.selenium.click_element(lookup_locator)
+        self.selenium.wait_until_page_contains_element(next_button, 60)
+        self.selenium.click_element(next_button)
+        self.selenium.wait_until_page_contains_element(related, 60)
+        self.selenium.scroll_element_into_view(related)
+        self.selenium.get_webelement(related).click()
+        self.selenium.click_element(option)
+        self.selenium.wait_until_page_contains_element(next_button, 60)
+        self.selenium.click_element(next_button)
+        self.salesforce.populate_field("Field Label", field_name)
+        self.salesforce.populate_field(
+            "Description", "This is a custom field generated during automation"
+        )
+        self.selenium.click_element(next_button)
+        self.selenium.click_element(next_button)
+        self.selenium.click_element(next_button)
+        self.selenium.click_element(save_button)
+        self.selenium.wait_until_location_contains(
+            "FieldsAndRelationships/view",
+            timeout=90,
+            message="Detail page did not load in 1 min",
+        )
+
+    @capture_screenshot_on_error
+    def is_field_present(self, field_name):
+        """Verifies and asserts the field got created """
+        self.selenium.wait_until_page_contains_element(search_button, 60)
+        self.selenium.get_webelement(search_button).send_keys(field_name)
+        self.selenium.get_webelement(search_button).send_keys(Keys.ENTER)
+        time.sleep(1)
+        self.salesforce.wait_until_loading_is_complete()
+        search_results = object_manager["search_result"].format(field_name)
+        assert (
+            len(self.selenium.get_webelements(search_results)) == 1
+        ), "Expected result count to be 1 but found {}".format(
+            len(self.selenium.get_webelements(search_results))
+        )
+
+    @capture_screenshot_on_error
+    def delete_custom_field(self):
+        """Deletes the added custom field"""
```

Comment:
There's not enough documentation for this keyword. It's not clear how or when you would use this. How does it know which field to delete? If this is only ever called from a certain page or dialog, you might want to mention that.

Reply:
Added